frontend/apps/marketing/src/components/snapshots/labSnapshot/LabSnapshot.tsx

```diff
@@ -0,0 +1,75 @@
+import {useMemo} from 'react';
+
+import Snapshot, {
+  SnapshotProps,
+  SnapshotItem,
+} from '@code-dot-org/component-library/cms/snapshot';
+
+export interface LabSnapshotProps extends Omit<SnapshotProps, 'items'> {
```

Comment:
Please, don't extend SnapshotProps here directly and only specify the props that will be passed to contentful component based on what you've specifed in definitions.
Current implementation will result in random contentful props being passed directly to component (e.g. isInExpEditorMode contentful internal prop, etc). Also, it misleads people re what props are actually present in the code.